styles/index.css

```diff
@@ -0,0 +1,92 @@
+* 
+body {
+    margin: 0;
+    font-family: 'Arial', sans-serif;
```

Comment:
we're gonna use koho for this so link the css koho file in the head and change "arial" to "Ko-Ho"